packages/purpose-process/src/services/validators.ts

```diff
@@ -89,7 +89,7 @@ export const assertEserviceMode = (
 };
 
 export const assertConsistentFreeOfCharge = (
-  isFreeOfCharge: boolean,
+  isFreeOfCharge: boolean | undefined,
```

Comment:
Shouldn't be needed? The field is mandatory in both update seeds
```suggestion
  isFreeOfCharge: boolean,
```

Reply:
Ok, since it is mandatory in the purpose model, I thought this validator would have been called either with the new value (optional from PATCH) or with the existing one - but at a practical level, there is no difference, since we only want to check the new value. Ok like this

Reply:
Actually we should always call this with either the new value of isFreeOfCharge, or the previously set one, so that we can cover the case where a user removes the freeOfCharge reason with the patch call